src/screens/settings/index.tsx

```diff
@@ -35,7 +36,7 @@ export default function SettingsScreen(props: SettingsScreenScreenProp) {
           </HeaderTitleContainer>
         )}
       />
-      <Container>
+      <Container contentContainerStyle={{ alignItems: 'center' }}>
```

Comment:
add showsVerticalScrollIndicator={flase} to remove scroll indicator when user scrolls 

```javascript
       <Container 
            contentContainerStyle={{ alignItems: 'center' }} 
            showsVerticalScrollIndicator={false}
        >
```